scripts/biomedical/usan/README.md

```diff
@@ -0,0 +1,67 @@
+# Importing the United States Adopted Names (USAN) data
+
+## Table of Contents
+
+- [Importing the United States Adopted Names (USAN) data](#importing-the-united-states-adopted-names-usan-data)
+  - [Table of Contents](#table-of-contents)
+  - [About the Dataset](#about-the-dataset)
+    - [Download Data](#download-data)
+    - [Overview](#overview)
+    - [Notes and Caveats](#notes-and-caveats)
+    - [License](#license)
+  - [About the import](#about-the-import)
+    - [Artifacts](#artifacts)
+      - [Scripts](#scripts)
+      - [Files](#files)
+  - [Example](#example)
+
+## About the Dataset
+
+[The United States Adopted Names](https://www.ama-assn.org/about/united-states-adopted-names/united-states-adopted-names-approved-stems) (USAN) provides meaningful designations for compounds that enhance prescription and patient safety. The listing of USAN stems represents common stems for which chemical and/or pharmacologic parameters have been established. These council-approved stems and their definitions are recommended for use in coining new nonproprietary drug names belonging to an established series of related agents.
+
+### Download Data
+
+The USAN stem data can be downloaded from their website [here](https://www.ama-assn.org/about/united-states-adopted-names/united-states-adopted-names-approved-stems). The data is in `.xlsx` format and had to be parsed first converted into a `.csv` format  (see [Notes and Caveats](#notes-and-caveats) for additional information on formatting) for further data preprocessing. For the convenience of users, we have provided a raw .csv version of the data in this repository because viewing the raw .xlsx version is unsupported in many systems.
+
+### Overview
+
+This directory stores the script used to clean and convert the United States Adopted Names data into a `.csv` format, which is ready for ingestion into the Data Commons knowledge graph alongside a `.tmcf` file that maps the `.csv` to the defined schema. In this import the data is ingested as [USAdoptedNameStem](https://datacommons.org/browser/USAdoptedNameStem) entities into the graph.
+
+The USAN Stem ID is mapped to other properties, namely, its name, definition, examples, stem type, word stem, specialization and year of creation.
+
+### Notes and Caveats
+
+The original format of the data was `.xlsx` and it was converted to a `.csv` file prior to ingestion into Data Commons. One of the key issues encountered during the import was working with a `.xlsx` file as it failed to be read by python due to different formatting. Hence, in order to preprocess the `.xlsx` file data, we first converted the file to a `.csv` format which was then read as a pandas dataframe in python. </br>
```

Comment:
This is not true. There is a command `pandas.read_excel` that can be used to read excel files into pandas. Either use that in your python script or add a conversion step to csv in your download script. https://pandas.pydata.org/docs/reference/api/pandas.read_excel.html